docs/api/matching-utilities.mdx

```diff
@@ -29,6 +39,67 @@ It will return a type guard function that returns `true` if _all_ of the provide
 
 Accepts the same inputs as `isAllOf` and will return a type guard function that returns `true` if at least one of the provided functions match.
 
+## `isAsyncThunkAction`
+
+A higher-order function that returns a type guard function that may be used to check whether an action was created by [`createAsyncThunk`](./createAsyncThunk).
+
+```ts title="isAsyncThunkAction usage"
+import { isAsyncThunkAction } from '@reduxjs/toolkit'
+import { request, request2 } from '@virtual/matchers'
+
+if (isAsyncThunkAction(request, request2)) {
+  // These are all async thunk actions, handle the case.
+}
+```
```

Comment:
```suggestion
```ts title="isAsyncThunkAction usage"
import { isAsyncThunkAction } from '@reduxjs/toolkit'
import { requestThunk1, requestThunk2 } from '@virtual/matchers'

const isARequestAction = isAsyncThunkAction(requestThunk1, requestThunk2);

function handleRequestAction(action: AnyAction) {
  if (isARequestAction(action)) {
    // action is either an action dispatched by `requestThunk1` or `requestThunk2`
  }
}
```
```

Reply:
the same pattern would apply to the others.